website/src/content/docs/general/support-policy.md

```diff
@@ -1,39 +1,82 @@
+- We do provide support for FluxCD setup and layout, as long as ClusterTool is used for the deployment.
+
+## Kubernetes Support Policy
+
+- We do not offer support for any component of kubernets that is not made by TrueCharts.
+
+## Helm Chart Support Policy
+
+- We offer support for all features available in our helm-charts, including all options in the common-chart.
+- We do not support external secret management outside of values.yaml
+
+:::note
+
+Some (common-chart) features might not work with all charts.
+
+:::
+
+## FluxCD Support Policy
+
+- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.
+
+## Structural rules
+
+- If its covered by support as outline above make a ticket in [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826).
+- [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) is not rendered on GitHub or any other platform.
+- Bugs must be reported on GitHub, but only after confirmation by the staff via a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826)
+- Anything not covered by the above Support Policy is not eligble for a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826). If you need assitance outside the scuoe of [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) make a thread in an appropiate [Discord Channel](https://discord.gg/tVsPTHWTtr).
 
 ## Expected Prior Knowledge
 
-- How to use a shell/console/terminal
-- Basic shell commands (`cp`, `ls`, `etc` and if you think `etc` is a command here, you'll have a problem)
-- Basic `kubectl` use (get pods, get svc, describe something etc)
-- Basic knowhow of Talos when you use it (things like: what is "Apply", what is a "machineconfig", what does "upgrade" do vs `k8s-upgrade`)
-- Basic Kubernetes knowhow (such as what is a "worker" node vs a "controlplane" node, what's "etcd", what's a "pod" vs a "container", etc.)
+- How to use a shell/console/terminal.
+- Basic shell commands (`cp`, `ls`, `etc`)
+- Basic Kubernetes know how (such as what is a "worker" node vs a "controlplane" node, what's "etcd", what's a "pod" vs a "container", etc).
 
 ## What's Explicitly Not Covered By Support:
 
-- Network configuration
-- CloudFlare proxying
-- Operating-Systems, outside of Talos deployed using ClusterTool
-- Helm 101 (e.g. "how do I install/upgrade/edit a Helm Chart?")
-- The software *contained* in our Helm Charts (for example "how do I configure Radarr?")
-- Cluster Operations (maintenance, fixing flaky pods, tracking non-deploying fluxcd helm-releases/customisations)
-- How to install/use `kubectl`/`talosctl`
+- Network/Router configuration.
+- CloudFlare proxying.
+- Operating-Systems, outside of Talos deployed using ClusterTool.
+- Helm 101 (e.g. "how do I install/upgrade/edit a Helm Chart?").
```

Comment:
```suggestion
- Helm 101 (e.g. "how do I install/upgrade/edit a Helm Chart?").
```

I think we can remove this, as we do provide docs for this and provide support if those things dont work.
It basically overlaps the "try yourself first" rule, while also sounding less friendly than we want to be